src/ApiService/ApiService/SingletonResources.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Microsoft.OneFuzz.Service;
+
+public interface ISingletonResources
+{
+
+    public System.Net.Http.HttpClient HttpClient { get; }
```

Comment:
there is [an issue](https://docs.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests) when using httpclietn as asingleton

Reply:
[this](https://docs.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests) seems to be the recommended way for creating httpclient instances